### PR TITLE
Improve pre-existing-vpc documentation

### PR DESCRIPTION
**Pull request by tpdownes**
- Correct description of input variable region to address misconception that a NAT and Router will be created (this module is read-only)
- Improve example and text in README
- Improve variable descriptions

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: modules/network/pre-existing-vpc/README.md [hunk 1]
```diff
@@ -16,14 +21,19 @@ sharing a single network module between deployment groups.
   source: modules/network/pre-existing-vpc
   settings:
   - project_id: $(vars.project_id)
```
Comment (rohitramu) [inline modules/network/pre-existing-vpc/README.md:23]: This probably shouldn't be a list, right?

File: modules/network/pre-existing-vpc/outputs.tf
Comment (rohitramu) [inline modules/network/pre-existing-vpc/outputs.tf:1]: Should there be a `nat_ips` output for this module to match the `vpc` module?

Comment (tpdownes) [reply modules/network/pre-existing-vpc/outputs.tf:1]: I've already documented this (and another) missing output value in a task. I'll communicate with you offline.

File: modules/network/pre-existing-vpc/README.md (hunk 1 above)
Comment (tpdownes) [reply modules/network/pre-existing-vpc/README.md]: Good catch. That was in original commit but I didn't notice.

**Review by rohitramu — APPROVED** [review]

**Pull request merged**